compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1409,19 +1425,15 @@ setDefaultGenericOpRootConfig(func::FuncOp entryPointFn,
         DispatchLoweringPassPipeline::CPUDefault);
   }
 
-  SmallVector<int64_t> minTileSizes = getMinTilingSizesForEachDim(
+  DistributionHeuristicConfig config;
```

Comment:
I feel like `distConfig` might be a clearer name given there can be other configs

Reply:
SGTM, thanks!